subprojects/docs/src/docs/userguide/tutorial/part5_gradle_inc_builds.adoc

```diff
@@ -0,0 +1,139 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[part5_gradle_inc_builds]]
+= Part 5: Exploring Incremental Builds
+
+Learn the basics of Gradle's incremental builds.
+
+****
+**In this section you will:**
+
+- Update Gradle Properties
+- Understand incremental builds
+****
+
+[[part5_begin]]
+== Step 0. Before you Begin
+
+1. You initialized your Java app in <<part1_gradle_init.adoc#part1_begin,part 1>>.
+2. You ran several tasks in <<part2_gradle_tasks#part2_begin,part 2>>.
+3. You learned about dependency management in <<part3_gradle_dep_man#part3_begin,part 3>>.
+4. You applied a plugin to your app in <<part4_gradle_plugins#part4_begin,part 4>>.
+
+== Step 1. Understanding Incremental Builds
+It's important to know that Gradle optimizes your build in a variety of ways.
+One such optimization is called _incremental builds_.
+
+An _incremental build_ is a build that avoids running tasks whose inputs and outputs did not change since the previous build, making the execution of such tasks unnecessary.
```

Comment:
Hmmm I'm not sure about the wording here - "whose inputs and outputs" - its more correct to say the inputs did not change and therefore the same outputs would be produced.

Would be good to also mention this applies to deterministic tasks only - something like "avoids running deterministic tasks whose inputs did not change and therefore would produce the same outputs that already exist" - may be good to check with others on the wording and have additional links to other docs

Reply:
Fixed. TY